### PR TITLE
support multiple path in GOPATH

### PR DESCRIPTION
**Pull request by wuyongzhi**
first path as scripts path

### CODE REVIEW

**Comment by albrow** [comment]
@wuyongzhi, looks good. Thank you! Could you please just add a quick comment to the code to explain why it's needed? I understand it since you provided some context in this PR but it's non-obvious for someone looking at just the code. As soon as you do I'll merge this with develop and do a patch release on master.

File: scripts.go [hunk 1]
```diff
@@ -11,6 +11,7 @@ import (
 	"path/filepath"
 	"text/template"
 	"time"
+	"string"
```
Comment (albrow) [inline scripts.go]: Is this a typo? I think it should be `strings`?

Comment (wuyongzhi) [inline scripts.go]: Yes, i am sorry.

**Comment by wuyongzhi** [comment]
i repull request  #12,  fixed some code and add some comments

**Pull request closed without merging**